app/views/user/_actions.haml

```diff
@@ -28,3 +28,7 @@
             %a.dropdown-item{ href: '#', data: { target: '#modal-ban', toggle: :modal } }
               %i.fa.fa-ban
               = t 'views.actions.ban'
+        - if current_user.has_role? :administrator
+          %a.dropdown-item{ href: "#{rails_admin_path}/user/#{user.id}", target: '_blank' }
```

Comment:
While those look fine now, couldn't we make it a bit cleaner?

Like, having a helper generate the full URL, being in parts the `rails_admin_path` but figuring the rest out itself? Working out the other URL fragment is the `resource` in some format + it's id

So having something like `rails_admin_path_for_resource(user)` would be swell, so we don't have a bunch of concatenation in the templates themselves!